pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -217,12 +217,12 @@ func (p *staticPolicy) updateCPUsToReuse(pod *v1.Pod, container *v1.Container, c
 
 func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Container) error {
 	if numCPUs := p.guaranteedCPUs(pod, container); numCPUs != 0 {
-		klog.Infof("[cpumanager] static policy: Allocate (pod: %s, container: %s)", pod.Name, container.Name)
+		klog.Infof("[cpumanager] static policy: Allocate (pod: %s, container: %s)", string(pod.UID), container.Name)
```

Comment:
Why this change (and the ones below)? I typically find it much harder to debug when looking at the UUID instead of the name.

Reply:
Reaction on this comment: https://github.com/kubernetes/kubernetes/pull/92967#discussion_r455429949 Maybe use this: https://github.com/kubernetes/kubernetes/blob/01df16aee8297041e32e0c7a2194a2749361d642/pkg/kubelet/util/format/pod.go#L32

Reply:
Alright, I applied mentioned function, so the result in logs will stem from this:
`fmt.Sprintf("%s_%s(%s)", podName, podNamespace, podUID`